nilearn/_utils/tags.py

```diff
@@ -107,14 +90,14 @@ def get_tag(estimator: BaseEstimator, tag: str) -> bool:
 
 
 def is_masker(estimator: BaseEstimator) -> bool:
+    if SKLEARN_LT_1_6:
+        return estimator._estimator_type == "masker"
     return get_tag(estimator, "masker")
 
 
-def is_multimasker(estimator: BaseEstimator) -> bool:
```

Comment:
we can try to just rely on the fact that multimasker are multi mixin instances for now